modules/wordpress-com-block-editor-iframe.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
Since it's a brand new file, what do you think about adding it to the PHPCS whitelist here?
https://github.com/Automattic/jetpack/blob/08a66d779ceb27fc5c98324ec4df8a0fa9f68f6f/bin/phpcs-whitelist.js#L13

Reply:
Done!